lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -4413,18 +4413,8 @@ namespace Js
             if (VirtualTableInfo<Js::LiteralStringWithPropertyStringPtr>::HasVirtualTable(index))
             {
                 LiteralStringWithPropertyStringPtr * str = (LiteralStringWithPropertyStringPtr *)index;
-                propertyString = str->GetPropertyString();
-                if (propertyString == nullptr)
-                {
-                    scriptContext->GetOrAddPropertyRecord(str->GetString(), str->GetLength(), &propertyRecord);
-                    propertyString = scriptContext->GetPropertyString(propertyRecord->GetPropertyId());
-                    str->SetPropertyString(propertyString);
-                }
-                else
-                {
-                    propertyRecord = propertyString->GetPropertyRecord();
-                }
-
+                propertyString = str->GetPropertyStringLazy();
```

Comment:
Why not also use `GetPropertyStringLazy` in `OP_GetElementI`?

Reply:
? sorry not sure I understood.

Reply:
Also looks like `JavascriptConversion::ToPropertyKey` could have some of it's code dealing with `LiteralStringWithPropertyStringPtr` replaced with a call to `GetPropertyStringLazy`

Reply:
`JavascriptConversion::ToPropertyKey` and similar places good idea. Could you please clarify -> `Why not also use GetPropertyStringLazy in OP_GetElementI` ?

Reply:
I was looking for where you had used `GetPropertyStringLazy`, and I saw some other places that `GetPropertyString` was used but you had not changed it to use `GetPropertyStringLazy`; is there a reason for that, or should we also update the other locations? Specifically for `OP_GetElementI` I was referring to https://github.com/Microsoft/ChakraCore/blob/master/lib/Runtime/Language/JavascriptOperators.cpp#L3697